modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -81,7 +81,7 @@ HTTP GET on resource 'http://jsonplaceholder.typicode.com:80/mybadrequest' faile
 +
 * `#[error.detailedDescription]` returns `HTTP GET on resource 'http://jsonplaceholder.typicode.com:80/mybadrequest' failed: not found (404).`
 * `#[error.errorType]` returns `HTTP:NOT_FOUND`
-* `#[error.cause]` returns `org.mule.extension.http.api.request.validator.ResponseValidatorTypedException`
+* `#[error.cause]` returns `org.mule.extension.http.api.request.validator.ResponseValidatorTypedException`. Note that if you print the return instance of `ResponseValidatorTypedException`, Mule will print the result of `toString`. When the error has an associated known HTTP status code (such as 404), the string representation of the `ResponseValidatorTypedException` instance is a human-readable description of the error.
```

Comment:
If you print the return instance of `ResponseValidatorTypedException`, Mule prints the result of `toString`. When the error has an associated known HTTP status code (such as 404), the string representation of the `ResponseValidatorTypedException` instance is a human-readable description of the error.